internal/pkg/externalplugins/plugins_test.go

```diff
@@ -97,7 +97,7 @@ func TestStartLoadConfig(t *testing.T) {
 	}
 
 	// Wait a moment.
-	time.Sleep(pullDuration + 1)
+	time.Sleep(pullDuration * 2)
```

Comment:
I don't like this change. We may add latch alike stable synchronize tool.

Reply:
I think it's OK to test. If a more stable synchronization tool is to be introduced, it may be necessary to make some changes to the code for testing purposes only.

Thanks for fix this invalid test!